### ISSUE TITLE
[BUG] Empty / Black infrared image

### ISSUE THREAD

**Issue by studentWUR** [luxonis/depthai#1009]
**Describe the bug**
All of my camera's are running as expected, but for one of them the rectified right infrared image, is completely black.
As visible in the image below 
![empty_image](https://user-images.githubusercontent.com/46886112/235098138-2ddb43c7-984e-4e67-92b3-8157df8c4797.JPG)
It seems that something is wrong with the IR right camera, but no errors appears.

I succesfully applied a factory reset using the [device manager](https://github.com/luxonis/depthai-python/tree/main/utilities), but the this did not help. 
Bootloader version: 0.0.24
Depthai version: 2.21.2.0

Any solutions for this problem? Or is this an hardware problem?

Thanks in advance.

**Comment by themarpe** [comment]
This might be a hardware issue CC: @Erol444 

Did you try using cam_test utility (under depthai-python) as well?

**Comment by studentWUR** [comment]
Sorry for my late reply I was on vacation. Yes this happens at cam_test as well. What should I do?

**Comment by Erol444** [comment]
@studentWUR could you share the full log from cam_test, and perhaps [enable debugging](https://docs.luxonis.com/projects/api/en/latest/tutorials/debugging/)?

**Comment by studentWUR** [comment]
Please let me know if this is sufficient

[log.txt](https://github.com/luxonis/depthai/files/11420398/log.txt)

**Comment by Erol444** [comment]
Hi @studentWUR , From the log it looks like it's working as expected, so that you are receiving about 14 FPS from all connected cameras. Could you confirm?
```
Connected cameras:
 -socket RGB   : IMX378 4056 x 3040 focus:auto  - COLOR
 -socket LEFT  : OV9282 1280 x  800 focus:fixed - MONO
 -socket RIGHT : OV9282 1280 x  800 focus:fixed - MONO
USB speed: UNKNOWN
IR drivers: [('LM3644', 2, 99)]
Cam:      rgb           left          right         camd     [host | capture timestamp]
FPS:  13.67| 21.76  13.40| 26.67  14.16| 26.67   0.00|  0.00
```

**Comment by studentWUR** [comment]
Yes, all of them have ~14 FPS.

**Comment by Erol444** [comment]
@studentWUR so images are being streamed, they are just black? If that's the case, please send an email to support@luxonis.com and we will send you a replacement camera, it might be a hardware issue (likely connection of the CCMs to the PCBA).

**Comment by studentWUR** [comment]
Yes, only for the right IR camera. I will contat support and close this issue. Thank you for your help
